cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -399,6 +402,16 @@ public static KafkaCluster fromCrd(Reconciliation reconciliation,
         if (kafkaClusterSpec.getTieredStorage() != null) {
             result.tieredStorage = kafkaClusterSpec.getTieredStorage();
         }
+
+        if (kafkaClusterSpec.getQuotas() != null) {
+            QuotasPlugin quotas = kafkaClusterSpec.getQuotas();
+
+            if (QuotasPluginUtils.isQuotasSpecCorrect(quotas, result.warningConditions)) {
+                result.quotas = quotas;
+                QuotasPluginUtils.ignoreQuotasConfigurationIfNeeded(result.configuration, result.quotas, result.warningConditions);
+            }
```

Comment:
Should we wrap this in a void method called "maybeSetQuotas", which takes quotas and result?

Reply:
Good point, I will change it a bit, so maybe it will be needed.
But if yes, I will wrap it as you suggested, thanks :)